source/manual/alerts/fastly-error-rate.html.md

```diff
@@ -37,7 +33,7 @@ The alert appears on `monitoring-1.management`. Collectd uses the Fastly API to
 
 To prove collectd is the problem, use this query in Kibana:
 
-`@source_host:monitoring-1.management AND @fields.syslog_program:collectd`
+`syslog_hostname:monitoring-1.management AND syslog_program:collectd`
 
 You will see many reports simlilar to:
```

Comment:
Would be nice to fix this spelling too while we're in this file :)

Reply:
Fixed. 👍